view/frontend/templates/script/sentry.phtml

```diff
@@ -35,7 +35,9 @@ if (typeof Sentry !== 'undefined') {
         environment: '<?= $block->escapeHtml(trim($block->getEnvironment())) ?>',
         integrations: [
             <?php if ($block->isTracingEnabled()): ?>
-                new Sentry.BrowserTracing(),
+            Sentry.browserTracingIntegration({
+                enableInp: true,
```

Comment:
Does enabling this raise your used quota?
If so we might want to make this configure it. If not, awesome! 🚀

Reply:
I have asked the sentry.io support.

Reply:
Should be fine: 

> when using enableInp - does this cause more events be recorded that count towards the quota ?

Support:

> No. Web Vitals (like INP) are measurements collected by in-browser Sentry SDKs, then added to the existing frontend transaction to provide more data about the Performance of the page. It enriches the transactions already coming in. There is more info on Web Vitals in our docs, here: https://docs.sentry.io/product/performance/web-vitals/web-vitals-concepts/.

Reply:
Awesome! I'll try and get it merged in tomorrow